Sources/MapboxNavigation/NavigationMapView+VanishingRouteLine.swift

```diff
@@ -146,10 +146,11 @@ extension NavigationMapView {
     }
     
     /**
-     Updates the route line appearance from the origin point to the indicated point
-     - parameter coordinate: current position of the puck
+     Updates the fractionTraveled along the route line from the origin point to the indicated point.
+     
+     - parameter coordinate: Current position of the user location.
      */
-    func updateTraveledRouteLine(_ coordinate: CLLocationCoordinate2D?) {
+    func updateFractionTraveled(_ coordinate: CLLocationCoordinate2D?) {
```

Comment:
A coordinate isn’t a fraction, so rename this method to `updateFractionTraveled(coordinate:)`, removing the `_`.